### DISCUSSION TITLE
Ultima PrimeNG RTL Support

### DISCUSSION THREAD

**Discussion by maqadeersaeed** [primefaces/community#894] [PrimeNG Templates] [answered]
We were using Ultima 13 which had RTL Support. But I am unable to see it in Ultima 17. Is RTL still supported in Ultima17 ? Please help ASAP

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Ultima received a huge update with v16 and became a next-gen template. We've removed the RTL support since custom CSS rules solely made it. We don't have plans to re-implement RTL support for now for the sake of simplicity and fewer style files.

**Reply by samerkh** [reply]
what do you mean by "custom css rules solely made it"?

**Reply by kashcode** [reply]
@cetincakiroglu Can you provide some examples? Thanks.
